Hamilton_Matthew/detail.html

```diff
@@ -0,0 +1,138 @@
+<%inherit file="app_base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="content_top">
+    <h1 class = "myh1">${ pagetitle }</h1>
+</%block>
+
+        <%block name="content_middle">
+            <ul>
+            <div class ="col-md-1">
+                    % for i in images:
+                        <li>  
+                            <img src = ${i} class = "left_pane_image" alt = ${i}>
```

Comment:
If you mimicked the naming convention of bootstrap and used hyphens rather than dashes it could be easier to distinguish what is a python method and what is just a CSS style. So you could change things like `left_pane_image` to `left-pane-image`.

Reply:
Fixed!